tests/test_validate_metadata.py

```diff
@@ -31,17 +31,18 @@
     validate_schema,
     CellMetadata,
     validate_collected_ontology_data,
+    validate_input_metadata,
 )
 
 
 class TestValidateMetadata(unittest.TestCase):
     def setup_metadata(self, args):
-        args_list = args.split(" ")
+        args_list = args.split(' ')
         args = create_parser().parse_args(args_list)
-        with open(args.convention, "r") as f:
+        with open(args.convention, 'r') as f:
```

Comment:
Read more (`r`) is default in [open()](https://docs.python.org/3/library/functions.html#open), so best to omit like we do for other defaults:

```suggestion
        with open(args.convention) as f:
```

Reply:
Updated in d58de2c.